docs/MO_DG/prepare_model/MO_Python_API.md

```diff
@@ -0,0 +1,72 @@
+## Model Optimizer Python API {#openvino_docs_MO_DG_prepare_model_MO_Python_API}
```

Comment:
I would suggest to have openvino_docs_MO_DG_MO_Python_API or openvino_docs_MO_DG_Python_API

Reply:
Changed to openvino_docs_MO_DG_Python_API.